nicegui/elements/input.py

```diff
@@ -58,7 +58,7 @@ def toggle_type(_):
             def find_autocompletion() -> Optional[str]:
                 if self.value:
                     for item in autocomplete:
-                        if item.startswith(self.value):
+                        if item.lower().startswith(self.value.lower()):
```

Comment:
Maybe you can `lower` the items only once in the constructor? That would save some cpu time...

Reply:
Never mind. I thought we could do

```py
autocomplete = [item.lower() for item in autocomplete]
```
But then we lose the casing completely. It's better done in the way you coded it!

Reply:
But you can move `needle = self.value.lower()` out of the fow-loop.